### ISSUE TITLE
[placeholder] Screen reader visual current item indicator

### ISSUE THREAD

**Issue by shubhi-raft** [raft-tech/TANF-app#488]
**Description:**
Follow-on from [PR](https://github.com/HHS/TANF-app/pull/106) for #119 
[Robert Jolly's comment](https://github.com/HHS/TANF-app/pull/106#discussion_r545962391):
>One request for a future story/release is when the usa-current class is added to primary nav item for the page, also add the ARIA attribute and value of aria-current="page" to the anchor element. This will supplement the experience for screen reader users who cannot see the visual current item indicator that folks who can see will receive.

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] _Outcome 1_
- [ ] _Outcome 2_
- [ ] _Outcome 3_
- [ ] Testing Checklist has been run and all tests pass

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] _Task 1_
- [ ] _Task 2_
- [ ] _Task 3_
- [ ] Run Testing Checklist and confirm all tests pass

**Notes:**
- Here's a [very good article from Léonie Watson's blog about using aria-current](https://tink.uk/using-the-aria-current-attribute/).

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

- _Doc 1_
- _Doc 2_

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_ 

- _Open Question 1_
- _Open Question 2_

**Comment by lfrohlich** [comment]
@reitermb  is this still an issue?

**Comment by reitermb** [comment]
> @reitermb is this still an issue?

Unless I'm missing it when inspecting the app (@jorgegonzalez please correct me if so!) I don't think this ever got implemented. Still a valid/recommended enhancement!

**Comment by jorgegonzalez** [comment]
Hmm I remember implementing this in likely a duplicate of this issue.

**Comment by jorgegonzalez** [comment]
![Screen Shot 2021-07-20 at 3 22 21 PM](https://user-images.githubusercontent.com/12901172/126383384-92110cc5-88f5-4ea5-977d-bf676c8080ea.png)

**Comment by reitermb** [comment]
@lfrohlich Glad I tagged Jorge, he's absolutely right, helps when I inspect the right component 🙃 This was duped and completed.

**Comment by amilash** [comment]
great! closing
